src/App.tsx

```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+import { Component } from 'react';
+import { Clock } from './components/Clock';
```

Comment:
You didn't save your Clock component. Try to push it to your repo

Reply:
Me to